scripts/get_openstack_plaintext_docs.sh

```diff
@@ -49,6 +50,25 @@ for project in "${os_projects[@]}"; do
     git switch stable/$OS_VERSION
     git pull origin stable/$OS_VERSION
 
+    # TODO(lpiwowar): Remove workarounds. Some of the documentations do not work with
+    # the feature of sphinx-build that allows generation of the docs in text format.
+    # List of issues:
```

Comment:
I am guessing you are planing to fix these as part of MR, or should these be translated to bugs/GH issues?

Reply:
No plan to fix these as part of the PR right now. The fixes would require some changes on the side of the individual projects. This workaround gives significant chunks of the documentation from the impacted projects. And even if we find and propose a fix, it would take some time until it gets merged.

I'm going to create an issue for now if there are now objections.

Reply:
https://github.com/openstack-lightspeed/rag-content/issues/9